### PR TITLE
Rename to DSCP

### PR DESCRIPTION
**Pull request by jonmattgray**
PR designed to merge shortly before/after renaming the repo to `dscp-identity-service`

Bump to `v1.0.0`
Change auth0 envs defaults to use newer `inteli` tenant

### CODE REVIEW

File: docker-compose.yaml
Comment (jonmattgray) [inline docker-compose.yaml:5]: need to wait until `vitalam-node` is renamed and built until we change or this repo's test workflow fails

same applies to `helm-lint` and `helm-test` in workflows which point to the `node` e.g. `ct lint --validate-maintainers false --charts helm/${{ needs.repo_ids.outputs.repo_name }}/ --chart-repos vitalam-node=https://digicatapult.github.io/vitalam-node/`

**Review by mattdean-digicatapult — COMMENTED** [review]
lgtm once the dps have been updated

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**
